### ISSUE TITLE
extract multiple containers extension from Droidium into separate project?

### ISSUE THREAD

**Issue by smiklosovic** [arquillian/arquillian-universe-bom#20]
Few people found multiple containers extension from Droidium useful in their test projects (1)

This extension is bundled into Droidium so even it can be referenced by gav from Maven Central, if somebody wants to modify it / repair some bugs and release it, whole Droidium has to be released as well.

I think this extension should be extracted into standalone project with its own development cycle.

(1) https://github.com/arquillian/arquillian-droidium/tree/master/droidium-container/arquillian-droidium-multiple-containers

**Comment by aslakknutsen** [comment]
Yeah, I think this is going into Core in the 1.2.x series. We're getting more and more 'containers' that have no problem living next to each other. e.g. Droidium, Cube, Chameleon..
